### ISSUE TITLE
Clone Operations Cause the Saving of Duplicate Tracks

### ISSUE THREAD

**Issue by DigitalProf** [BYU-ARCLITE/Ayamel-Examples#732]
Using this content object: https://ayamelbeta.byu.edu/content/579 

The "Clone with Translation" option saved two copies of the new English track. The problem could well be that one copy is saved upon creation, but the appropriate flag is not set as to existing track. Thus, when the user clicks on "Save Track", another save is created.
![image](https://cloud.githubusercontent.com/assets/7268293/12868023/0c9e9e98-ccba-11e5-95f5-d0d5c1266ba3.png)

**Comment by DigitalProf** [comment]
This was previously documented as "Clone with Translation' Saving Two New Tracks" which was on the Beta Site. It turns out that this happens whenever a track is cloned using any of the clone operations, even on the Production Site.

**Comment by gliese1337** [comment]
I am not able to replicate this. I've tried cloning tracks on my local development copy, and cloning a track on that same Dreyfus video on the main site, and it appears to be working correctly. I've traced the process from clicking the "clone track" menu option every step of the way through saving new  & edited tracks and displaying them on the content page, and at no point are any duplicates created.

**Comment by DigitalProf** [comment]
Weird... I will clear cache (again) and do it again, recording the screen as I go.

Sent from my iPhone

On Feb 10, 2016, at 3:24 PM, Logan Kearsley <notifications@github.com<mailto:notifications@github.com>> wrote:


I am not able to replicate this. I've tried cloning tracks on my local development copy, and cloning a track on that same Dreyfus video on the main site, and it appears to be working correctly. I've traced the process from clicking the "clone track" menu option every step of the way through saving new & edited tracks and displaying them on the content page, and at no point are any duplicates created.

-
